app/controllers/api/services_controller.rb

```diff
@@ -75,24 +75,43 @@ def destroy
 
   private
 
+  attr_reader :service
+
+  def integration_settings_result
+    ApiIntegration::SettingsResult.new(service: service, proxy: service.proxy)
+  end
+
+  def create_params
```

Comment:
I know that this is very bad but I encountered conflicts with this during this same morning :scream: And I don't have time to fix it properly for today.

Reply:
Ah you mean repeating it :)

Yes you can DRY with `user_attribute_names` later

Reply:
In fact it should be done properly in [THREESCALE-3363](https://issues.jboss.org/browse/THREESCALE-3363), but that is after 2.7